tabs-tabs-tabs/tabs.js

```diff
@@ -70,5 +70,49 @@ document.addEventListener("click", function(e) {
     });
   }
 
+  else if (e.target.id === "tabs-add-zoom") {
+      callOnActiveTab((tab) => {
```

Comment:
The rest of this file uses 2-space tabs, could you adjust the new code to be consistent with that?